radix-engine/src/system/system_modules/transaction_runtime/module.rs

```diff
@@ -14,7 +14,7 @@ pub struct Event {
 
 impl Event {
     pub fn len(&self) -> usize {
-        self.type_identifier.0.get_length()
+        self.type_identifier.len() + self.payload.len() + 4
```

Comment:
What's the +4 for?

Reply:
This is to account for event flags.

Reply:
please consider a well-named const
_(which also creates a perfect place for putting a rustdoc on, with more detailed explanation)_